module/kvmfr.c

```diff
@@ -277,6 +283,11 @@ static int device_mmap(struct file * filp, struct vm_area_struct * vma)
 
   switch (kdev->type)
   {
+    case KVMFR_TYPE_PCI:
+    {
+      unsigned long pfn = virt_to_phys(kdev->addr + offset) >> PAGE_SHIFT;
+      return remap_pfn_range(vma, vma->vm_start, pfn, size, vma->vm_page_prot);
+    }
```

Comment:
nit: whitespace after the `}` please

Reply:
I missed the shift, ignore `phys_add_t` change.

Reply:
Fixed.